docs/design/architectural-overview.md

```diff
@@ -0,0 +1,168 @@
+# Kuadrant Architectural Overview [Draft]
+
+## Overview
+
+It is important to note that Kuadrant is not in itself a gateway provider. Kuadrant provides a set of valuable policy APIs that enhance [Gateway API](https://github.com/kubernetes-sigs/gateway-api) via its defined [policy attachment](https://gateway-api.sigs.k8s.io/references/policy-attachment/) extension point. The policy APIs are reconciled by a set of policy controllers and enforced via integration at different points to configure, enhance and secure the application connectivity provided via Gateway API and the underlying gateway provider. 
+These policy extensions are focused around areas such as DNS management supporting global load balancing and health checks, alongside service protection specific APIs such as rate limiting and auth. Kuadrant also integrates with [OCM (Open Cluster Manager)](https://open-cluster-management.io/) as a multi-cluster control plane to enable defining and distributing Gateways across multiple clusters, providing load balancing and tls management for these distributed gateways alongside the single view and single definition of gateways from the OCM management hub. 
+
+
+## Key Architectural Areas
+
+* The Kuadrant architecture is spread across a control plane and also a data plane. Kuadrant can work in both a single and multi-cluster context. 
+Currently in order for all APIs to work in a single or multi-cluster context you need to have OCM installed.  While this may change in the future, this approach allows us to start with a single cluster and seamlessly scale as more clusters are added.
+* The control plane is where policies are exposed and expressed as kubernetes APIs and reconciled by the Kuadrant policy controllers. 
+* The data plane is where Kuadrant's service protection components, configured by the  control plane policies, are enforced within the gateway instance as part of the request flow. 
+
+## 1000m Architecture
+
+![](./images/1000m-overview.png)
+
+**Note:**
```

Comment:
What make you feel that such transient warning was to be put in the architecture document?

Reply:
Yes this is fair. Really this should be something reflected in a different document (IE some roadmap)